services/ui-src/src/containers/NewAmendment.js

```diff
@@ -45,6 +45,15 @@ export default function NewAmendment({ fileUpload }) {
 
     if (!validateFileAttachment(file)) return;
 
+    var validExt = ".png, .gif, .jpeg, .jpg";
```

Comment:
@ayoneeyee as mentioned in the issue description. we should use the allowed file types as defined in the macstack pull request which resolved this issue within their repo
https://github.com/CMSgov/macstack-spa-submission-form/pull/227/files